tests/config/block-fixtures.ts

```diff
@@ -0,0 +1,94 @@
+import type { Page, Locator } from '@playwright/test';
+
+export class BlockFixtures {
+  constructor(public readonly page: Page) {
+    //
+  }
+
+  /**
+   * Adds an image to the first image slot in a block by selecting it from the media library.
+   *
+   * @param imageLabel - The aria-label of the image to select from the media library
+   *
+   * @usage - await blockFixtures.addImageToAvailableSlot('My Image');
+   */
+  async addImageToAvailableSlot(imageLabel: string) {
+    await this.page
+      .locator('iframe[name="editor-canvas"]')
+      .contentFrame()
+      .getByRole('button', { name: 'Select Image' })
+      .first()
+      .click();
+
+    // Get the aria-checked attribute of the image
+    const isImageSelected = await this.page.getByLabel(imageLabel).getAttribute('aria-checked');
+
+    // If the image is not already selected, select it
+    if (isImageSelected === 'false') {
+      await this.page.getByLabel(imageLabel).click();
+    }
+
+    // Press the Select button to use the image
+    await this.page.getByRole('button', { name: 'Select', exact: true }).click();
+  }
+
+  /**
+   * Adds an image to the second image slot in a block by selecting it from the media library.
+   *
+   * @param imageLabel - The aria-label of the image to select from the media library
+   *
+   * @usage - await blockFixtures.addImageToSecondSlot('My Image');
+   */
+  async addImageToSecondSlot(imageLabel: string) {
+    await this.page
+      .locator('iframe[name="editor-canvas"]')
+      .contentFrame()
+      .getByRole('button', { name: 'Select Image' })
+      .nth(1)
+      .click();
+
```

Comment:
at this point in time, we can't guarantee that the correct media library tab is selected, so it's best to ensure that

```suggestion

    await this.page.locator('#menu-item-browse').click();

```